api/tests/routes/pro/signup_pro_V2_test.py

```diff
@@ -59,6 +64,29 @@ def test_when_user_data_is_valid(self, client):
         assert mails_testing.outbox[0]["To"] == user.email
         assert mails_testing.outbox[0]["template"] == asdict(TransactionalEmail.EMAIL_VALIDATION_TO_PRO.value)
 
+    @override_features(WIP_ENABLE_NEW_NAV_AB_TEST=True)
+    def test_user_pro_new_nav_bar_ab_(self, client):
+        data = BASE_DATA_PRO.copy()
+        emails = []
+        for _ in range(2):
+            data["email"] = "".join(random.choices(string.ascii_letters, k=12)).lower() + "@test.com"
+            client.post("/v2/users/signup/pro", json=data)
+            emails.append(data["email"])
```

Comment:
[détail] J'ai l'impression qu'on s'embête un peu pour rien. :)

```suggestion
        emails = ["user1@example.com", "user2@example.com"]
        for email in emails:
            client.post("/v2/users/signup/pro", json=data | {"email": email})
```